pkg/asset/agent/installconfig.go

```diff
@@ -215,9 +215,15 @@ func (a *OptionalInstallConfig) validateControlPlaneConfiguration(installConfig
 	var fieldPath *field.Path
 
 	if installConfig.ControlPlane != nil {
-		if *installConfig.ControlPlane.Replicas < 1 || *installConfig.ControlPlane.Replicas > 5 || *installConfig.ControlPlane.Replicas == 2 {
+		if *installConfig.ControlPlane.Replicas < 1 || *installConfig.ControlPlane.Replicas > 5 || (installConfig.Arbiter == nil && *installConfig.ControlPlane.Replicas == 2) {
 			fieldPath = field.NewPath("ControlPlane", "Replicas")
-			allErrs = append(allErrs, field.Invalid(fieldPath, installConfig.ControlPlane.Replicas, fmt.Sprintf("ControlPlane.Replicas can only be set to 5, 4, 3, or 1. Found %v", *installConfig.ControlPlane.Replicas)))
+			// Make sure users get an accurate error message while feature is in TechPreview
+			// TODO: Remove once HighlyAvailableArbiterMode moves out of TechPreviewNoUpgrade and just use `"between 1 and 5"`
+			supportedControlPlaneRange := "to 5, 4, 3, or 1"
+			if installConfig.FeatureSet == configv1.TechPreviewNoUpgrade {
```

Comment:
Note that ABI doesn't actually support FeatureSet yet - it fails with an error if you specify TechPreviewNoUpgrade because it doesn't have a mechanism to pass that on to assisted-service.
https://issues.redhat.com/browse/AGENT-554

This is probably the right thing to do though, as it means we will get the right behaviour if somebody implements that.

Reply:
Other than having the capacity to work on it, are there technical reasons this would be a problem to implement? If it's wiring things up, I might have some cycles to push it through if we want this for ABI as TechPreview.

Reply:
It's not technically difficult, it just hasn't reached the top of the priority list. I'd be more than happy for you to pick up that ticket. Let me or @rwsu know if you need any help.

Reply:
Sounds good, I'll keep you posted, thanks!